src/redux/thunks/index.js

```diff
@@ -4,10 +4,15 @@ export const axiosInstance = axios.create({
   baseURL: 'https://ah-jawans-backend.herokuapp.com/api/',
 });
 
-// eslint-disable-next-line max-len
-const postDataThunk = (method, endpoint, actionCreator, data) => (dispatch) => axiosInstance[method](endpoint, data)
+const postDataThunk = (
+  method,
+  endpoint,
+  actionCreator,
+  data,
+) => (dispatch) => axiosInstance[method](endpoint, data)
   .then((response) => {
     dispatch(actionCreator(response.data));
+    localStorage.setItem('token', JSON.stringify(data.data));
```

Comment:
you can remove this line , i am storing the localstorage token in my PR. this is not the correct place to do it.